sci-visualization/mantid/mantid-3.3.0.ebuild

```diff
@@ -85,7 +84,45 @@ src_configure() {
 }
 
 src_test() {
+	pushd "${BUILD_DIR}" # For some reason this is not done automatically...
```

Comment:
If you are not using `cmake-utils_src_test`, you will have to do that manually.
You could try to put the tests in the `myctestargs` array.